ui/raidboss/data/02-arr/alliance/the_world_of_darkness.ts

```diff
@@ -0,0 +1,103 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  sullenDebuff?: boolean;
+  irefulDebuff?: boolean;
+}
+
+// TODO:
+//  Angra Mainyu
+//    Add Level 100 Flare
+//    Add Level 150 Doom
+//    Add Roulette?
+//    Add info text for add spawns?
+//  Five-Headed Dragon
+//  Howling Atomos
+//  Cerberus
+//  Cloud of Darkness
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheWorldOfDarkness,
+  triggers: [
+    {
+      id: 'Angra Mainyu Gain Sullen',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: '27c' }),
+      condition: Conditions.targetIsYou(),
+      run: (data) => data.sullenDebuff = true,
+    },
+    {
+      id: 'Angra Mainyu Lose Sullen',
+      type: 'LosesEffect',
+      netRegex: NetRegexes.losesEffect({ effectId: '27c' }),
+      condition: Conditions.targetIsYou(),
+      run: (data) => data.sullenDebuff = false,
+    },
+    {
+      id: 'Angra Mainyu Gain Ireful',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: '27d' }),
+      condition: Conditions.targetIsYou(),
+      run: (data) => data.irefulDebuff = true,
+    },
+    {
+      id: 'Angra Mainyu Lose Ireful',
+      type: 'LosesEffect',
+      netRegex: NetRegexes.losesEffect({ effectId: '27d' }),
+      condition: Conditions.targetIsYou(),
+      run: (data) => data.irefulDebuff = false,
+    },
+    {
+      id: 'Angra Mainyu Double Vision',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: 'CC8', source: 'Angra Mainyu', capture: false }),
+      alertText: (data, _matches, output) => {
+        if (data.sullenDebuff) {
+          // Stand in red half to switch to Ireful
+          return output.red!();
+        } else if (data.irefulDebuff) {
+          // Stand in white half to switch to Sullen
+          return output.white!();
+        }
+        return output.vision!();
```

Comment:
Would it be better to say nothing here if you don't have a debuff? I'm not sure that "Double Vision" is really actionable.

Reply:
So the main reason I had this was for testing to make sure things were actually appearing. In my log, a lot of players die and then don't get any output without it because they don't have the debuff anymore.

My main concern with printing nothing is that it would potentially feel weird like "aren't i supposed to do something here?" I guess we could also code it so that for DPS it starts with the rear and for Tank it starts with the front side if you have no debuff?